Formula/orientdb.rb

```diff
@@ -5,6 +5,11 @@ class Orientdb < Formula
   sha256 "e548b40cb974cbd8a014ee6dcae6e0cd1fc5d8a6473166d18c08fe1f3bbbb077"
   license "Apache-2.0"
 
+  livecheck do
+    url "https://orientdb.org/download"
+    regex(%r{href=.*orientdb3/releases/(\d+(?:\.\d+)+)/}i)
```

Comment:
```suggestion
    regex(/href=.*?orientdb[._-]v?(\d+(?:\.\d+)+)\.zip/i)
```

Since the version is found in the filename of the `stable` archive file that's linked on the download page, it's preferable to obtain the version number from that. My suggested regex above may also be a bit more reliable over time, as it won't need to be updated if/when the software reaches 4.x.